app-misc/i2bits/Manifest

```diff
@@ -1 +1 @@
-DIST i2bits-0.1.0.tar.gz 2649 BLAKE2B 050e7eddadc4e8b4e1da50128b735adbe6fbace35a06709fd2f31db3e8ffb672b3d51eced0ecb9c87c3566d32c9aac7a431c641e53597ee55ae431328dfbee48 SHA512 9f7b2fe8ca66740cae081c3f74f0c5d67d956c8cd2b5f39cb7306e58a8f75d6d5848dd640e7e514638dda6f3d1efe5f292605dc4fddffe5e530bbd5f173a1db9
+DIST i2bits-0.1.0-bb.tar.gz 3007 BLAKE2B 51f866c7d91a1aedf88f0e30bb4847fe659b6abbaea1972a11d4d91cdb053a2b2add2fbe81a9eb9e1246e2373a332ed012ea80cd247548dc34106075d0a53d25 SHA512 ccd5518e05c13727a1698177d485d26ac2f7adc8e49350a9395df28cb5431b74ddf4603ef242ad5f7703e8184346871b70fc03afc506a8d65df9db010c1cb7b4
```

Comment:
If you're changing SRC_URI such that the hash/manifest changes, please include a summary of the diff of the tarballs in the commit message, per https://devmanual.gentoo.org/general-concepts/manifest/index.html#updating-manifest-files.

Reply:
Please resubmit. I've reverted it in e14722429f99138e7ea6426f427a8dfa464bf371.